apps/src/hooks/use-url-sync.ts

```diff
@@ -280,7 +267,8 @@ export const useUrlSync = () => {
 		opacity,
 		dataset,
 		mapCoordinates,
-		isInitialized
```

Comment:
(*formatting:* another example of why it's so useful to have trailing comas all the time. This would be one line otherwise)